### PR TITLE
feat(icons): introduce package manager

### PR DESCRIPTION
**Pull request by AugustinMauroy**
## Description

We will need these icons 

## Validation

It's shouldn't break trademark policy of theses projects

### Check List

- [x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run `npm run format` to ensure the code follows the style guide.
- [x] I have run `npm run test` to check if all tests are passing.
- [x] I have run `npx turbo build` to check if the website builds without errors.

### CODE REVIEW

File: apps/site/components/.DS_Store
Comment (ovflowd) [inline apps/site/components/.DS_Store:1]: Can you uncommit this file?

File: apps/site/components/Icons/PackageManager/Yarn.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import type { FC, SVGProps } from 'react';
+
+/**
+ * @see https://github.com/yarnpkg/assets/blob/master/yarn-kitten-circle.svg
+ */
+const YarnIcon: FC<SVGProps<SVGSVGElement>> = props => (
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 518 518" {...props}>
```
Comment (ovflowd) [inline apps/site/components/Icons/PackageManager/Yarn.tsx]: Can you add a default width height here too?

File: apps/site/components/Icons/PackageManager/Npm.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import type { FC, SVGProps } from 'react';
+
+/**
+ * @see https://github.com/npm/logos/blob/master/npm%20square/n.svg
+ */
+const NpmIcon: FC<SVGProps<SVGSVGElement>> = props => (
+  <svg
+    width="800px"
```
Comment (ovflowd) [inline apps/site/components/Icons/PackageManager/Npm.tsx]: These default width heights are gigantic, check the other icons they should be smaller no?

File: apps/site/components/.DS_Store
Comment (AugustinMauroy) [reply apps/site/components/.DS_Store:1]: What is this ?

**Comment by AugustinMauroy** [comment]
for size I use `32px`x`32px`

File: apps/site/components/.DS_Store
Comment (ovflowd) [reply apps/site/components/.DS_Store:1]: Comes from macOS

Comment (AugustinMauroy) [reply apps/site/components/.DS_Store:1]: shouldn't we put in `.gitignore` ?

Comment (ovflowd) [reply apps/site/components/.DS_Store:1]: No. You simply do not commit it lol. Gitignore is for project files not OS files.

You can add that to your global gitignore file on your home folder.

Comment (AugustinMauroy) [reply apps/site/components/.DS_Store:1]: > You can add that to your global gitignore file on your home folder.

OOOHH I'll do that thanks !

File: apps/site/components/Icons/PackageManager/Npm.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import type { FC, SVGProps } from 'react';
+
+/**
```
Comment (ovflowd) [inline apps/site/components/Icons/PackageManager/Npm.tsx]: Why these @see statements? Are they really needed?

Comment (ovflowd) [reply apps/site/components/Icons/PackageManager/Npm.tsx]: We don't have'em in any of the other components, so let's keep it standard

**Review by ovflowd — APPROVED** [review]
LGTM!

**Pull request merged**